docs/deploy/howto-deploy-aks.md

```diff
@@ -514,34 +514,80 @@ Command line argument key-value pairs can be specified with:
 
 ### AKS
 
-All of microservices of Industrial IoT solution are deployed to an AKS Kubernetes cluster. The deployment
-creates `industrial-iot` namespace where all microservices are running. To provide connection details for
-all Azure resources created by `Microsoft.Azure.IIoT.Deployment`, we created `industrial-iot-env` secret,
-which is then consumed by deployments.
+All of microservices of Industrial IoT solution are deployed to an AKS Kubernetes cluster.
+
```

Comment:
All cloud microservices of Industrial IoT platform...

Reply:
Done.